lib/fastlane/plugin/ddg_apple_automation/actions/asana_add_comment_action.rb

```diff
@@ -25,10 +24,11 @@ def self.run(params)
         task_id = Fastlane::Actions::AsanaExtractTaskIdAction.run(task_url: task_url) if task_url
 
         if template_name.to_s.empty?
-          text = "#{comment}\n\nWorkflow URL: #{workflow_url}"
+          text = "#{comment}\n\nWorkflow URL: #{ENV.fetch('WORKFLOW_URL')}"
```

Comment:
Please mind that `ENV.fetch('WORKFLOW_URL')` will raise KeyError when a given variable isn't present. To prevent that, provide a default value that will be used when the variable is not found, e.g. `ENV.fetch('WORKFLOW_URL', '')`